tests/data/fixtures/db.yaml

```diff
@@ -376,8 +376,8 @@
     last_login: 2020-01-25 14:23:16.175169+00:00
     is_superuser: false
     email: public@rev.com
-    first_name: ''
```

Comment:
revert this and w/ yaml formatting disabled make the edits manually

Reply:
or, let me know how you have yaml formatting enabled in vscode and I can add it to my editor